<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crearmesas3</class>
 <widget class="QWidget" name="crearmesas3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cantidad">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);
</string>
   </property>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);
</string>
   </property>
   <property name="text">
    <string>Personas en Mesa: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="activarMesa">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Activar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numeroMesa">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 24pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);
</string>
   </property>
   <property name="text">
    <string>MANEJAR MESAS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="crearMesita">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>350</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);
</string>
   </property>
   <property name="text">
    <string>Numero de Mesa: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);
</string>
   </property>
   <property name="text">
    <string>Numero de Mesa: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 24pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);
</string>
   </property>
   <property name="text">
    <string>CREAR MESAS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="desactivarMesa">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>240</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Desactivar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
